docs/language.rst

```diff
@@ -342,7 +342,9 @@ Instances of an entity are created with a constructor statement
 A constructor can assign values to any of the properties (attributes or relations) of the entity. It can also leave the properties unassigned.
 For attributes with default values, the constructor is the only place where the defaults can be overridden.
 
-Values can be assigned to the remaining properties as if they are variables. To relations with a higher arity, multiple values can be assigned
+Values can be assigned to the remaining properties as if they are variables. To relations with a higher arity, multiple values can be assigned.
+Additionally, `null` can be assigned to relations with a lower arity of 0 to indicate explicitly that the model will not assign
```

Comment:
should we allow it for [0:]?

Reply:
I wasn't sure of this myself. In any other language I'd say no because assigning `null` to a list variable does not make sense. But a `[0:]` relation is not just a list. We could say such a relation is either `null` or a collection of references to other entities. This actually corresponds to the behavior of leaving it unassigned I think. Iirc `instance.relation is defined == true` in that case. I'll test this tomorrow, I think `is defined` offers a good guideline for consistency. If my assumption about this is correct, I think we could definitely allow it for these relations (though it might be less useful than for `[0:1]`) Otherwise I'd disallow it.
What's your opinion?

Reply:
I hadn't really considered the possibility, so I'm a bit hesitant to jump to hasty conclusions. 

Forcing a list to be empty is a bit weird. But useful for the same reason as on optionals.  
So I hand't considered it, but beautifully consistent.

Reply:
I've verified `is defined` behavior is as I assumed above. I'm in favor of allowing it for `[0:]` but I don't feel very strongly about this. What do you say, leave it or revert it?

Reply:
Leave it